intake-catalogs/hydro.yaml

```diff
@@ -60,3 +60,46 @@ sources:
     args:
       urlpath: 'gs://pangeo-data/hydrosheds/dem.vrt'
       chunks: {'y': 6000, 'x': 6000}
+
+  soil_grids_single_level:
+    description: Global gridded soil information in COG format
+    parameters:
+      variable:
+        description: soil variable
+        type: str
+        default: TAXNWRB
+    metadata:
+      url: https://soilgrids.org/
+      tags: Soil
+    driver: rasterio
+    args:
+      urlpath: "gs://pangeo-data/soilgrids/{{ variable }}_250m.tif"
+      chunks: {'y': 5120, 'x': 5120}
+      storage_options: {'anon': True}  # TODO: remove, see GH#61
```

Comment:
@rabernat and @davidbrochart, wondering if you have suggestions on how to do this. This catalog loads fine in ocean.pangeo.io and fails on travis with the following error:

```
E   rasterio._err.CPLE_AppDefinedError: Fetching OAuth2 access code from auth code failed.
```

The bucket is currently public so I'm not sure why I need to do any auth here. Thoughts?

Reply:
We have a similar issue in #59.

Reply:
Actually this is a different issue. This one is more similar to https://github.com/intake/intake-xarray/issues/31. I made a PR here: https://github.com/jhamman/pangeo-datastore/pull/1